addons/nuxeo-multi-tenant/nuxeo-multi-tenant-core/src/test/java/org/nuxeo/ecm/multi/tenant/TestMultiTenantDirectories.java

```diff
@@ -55,7 +56,7 @@
  * @since 10.2
  */
 @RunWith(FeaturesRunner.class)
-@Features({ DirectoryFeature.class, PlatformFeature.class })
+@Features({ DirectoryFeature.class, NuxeoLoginFeature.class, PlatformFeature.class })
```

Comment:
Why do you need the NuxeoLoginFeature in this tests and the other multitenant tests that didn't deploy the login component?

Reply:
Because tests explicitly use `Framework#loginAs` and expect it to leverage the multi tenant user manager. If the feature is not present we will have an exception (NPE) because `loginPluginManager` is not deployed. This is due to test contrib which deploys `NuxeoLoginModule`.

In the end, I will clean the contrib and leave the `NuxeoLoginFeature`.

Reply:
OK, tricky :smiley: